xrpl4j-core/src/main/resources/definitions.json

```diff
@@ -140,40 +141,40 @@
     [
       "LedgerEntry",
       {
-        "nth": 1,
+        "nth": 257,
         "isVLEncoded": false,
         "isSerialized": false,
-        "isSigningField": true,
+        "isSigningField": false,
         "type": "LedgerEntry"
       }
     ],
     [
       "Transaction",
       {
-        "nth": 1,
+        "nth": 257,
         "isVLEncoded": false,
         "isSerialized": false,
-        "isSigningField": true,
+        "isSigningField": false,
```

Comment:
Why are these changing?

Reply:
I generally run [xrpl-codec-gen](https://github.com/RichardAH/xrpl-codec-gen) to generate this file whenever there are new transactions or fields. Mayukha's [recent commit](https://github.com/RichardAH/xrpl-codec-gen/commit/d3691947bfcc334a5c1a9322ad2e53626adcc5ea) changes these fields `isSigningField` value to align with her rippled PR, which explains these changes.